### PR TITLE
Caption component

### LINKED ISSUE #57: Put Caption component into Psammead
**Is your feature request related to a problem? Please describe.**
Corresponding 'pull from Psammead' issue in Simorgh: https://github.com/BBC-News/simorgh/issues/1002.

**Describe the solution you'd like**
A PR in Psammead, which adds Caption component.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Unit tests (if any) will be transferred from Simorgh.

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #57 

Caption component

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-caption/src/index.test.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import Caption from './index';
```
Comment (ChrisBAshton) [inline packages/components/psammead-caption/src/index.test.jsx]:
```suggestion
import Caption from '.';
```

File: packages/components/psammead-caption/src/index.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import InlineLink from '@bbc/psammead-inline-link';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import Caption from './index';
```
Comment (ChrisBAshton) [inline packages/components/psammead-caption/src/index.stories.jsx]:
```suggestion
import Caption from '.';
```

File: packages/components/psammead-caption/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# psammead-caption &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-caption.svg)](https://www.npmjs.com/package/@bbc/psammead-caption) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description	
+The `psammead-caption` component is a styled `figcaption` element.
+
+## Usage 
+
+`psammead-caption` can be used together with `psammead-visually-hidden-text` in this way. This visually hidden text is to give context to users of Assistive Technology.
+
+```
```
Comment (ChrisBAshton) [inline packages/components/psammead-caption/README.md]: \```jsx

**Review by ChrisBAshton — COMMENTED** [review]
A couple of formatting things. 👍 looks good

**Review by bcmn — APPROVED** [review]
Looks good, & tested locally.

**Review by ChrisBAshton — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**